### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by bobbygondola**
FIX! TLContainer grows if not PX. words too big for Card

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by corbinrobb — APPROVED** [review]

File: src/components/Navigation/Navigation.js [hunk 1]
```diff
@@ -40,6 +42,9 @@ const Navigation = props => {
             {/* <Link to="/new-topic"> */}
```
Comment (ryan-hamblin) [inline src/components/Navigation/Navigation.js:42]: Please remove any commented out code from main. It doesn't have a place in a production environment.

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -5,13 +5,15 @@ import {
...
 } from '../../state/actions/displayModalAction';
+import { useOktaAuth } from '@okta/okta-react';
 import 'antd/dist/antd.css';
 
 import { Layout, Menu } from 'antd';
 
 const { Header, Content, Footer } = Layout;
 
 const Navigation = props => {
+  const { authState, authService } = useOktaAuth();
```
Comment (ryan-hamblin) [inline src/components/Navigation/Navigation.js:16]: You've destructured `authState` however you're not using it. Are you sure you need to include this to achive the logout functionality for the app?

File: src/components/TopicDetails/RenderTopicDetails.js [hunk 3]
```diff
@@ -41,20 +41,23 @@ function RenderTopicDetails(props) {
...
           <h2>{topicDetailsInfo.title}</h2>
+          <Dropdown overlay={menu}>
+            <Button onClick={e => e.preventDefault()}>Select</Button>
+          </Dropdown>
           <h2>Members: </h2>
-          {topicDetailsInfo.members.map(member => (
-            <span>{`${member.name}, `}</span>
-          ))}
+          {topicDetailsInfo.members.map(member => {
+            console.log(member);
```
Comment (ryan-hamblin) [inline src/components/TopicDetails/RenderTopicDetails.js:51]: Please remove any console.logs from main. Print statements don't have a place in production code.

File: src/components/TopicsList/RenderTopicsList.js [hunk 4]
```diff
@@ -53,9 +53,10 @@ const RenderTopicsList = props => {
   return (
     <>
       <div className="topics-list">
-        <Button onClick={displayTopicsICreated}>Topics I Created</Button>
-        <Button onClick={displayTopicsIJoined}>Topics I Joined</Button>
-        <h2>Topics</h2>
+        <div className="topic-buttons">
+          <Button onClick={displayTopicsICreated}>Topics I've Created</Button>
```
Comment (ryan-hamblin) [inline src/components/TopicsList/RenderTopicsList.js:57]: This click (although I love its name) could actually be a single function. 

`onClick={(e) => displayTopics(e, data)}`

`const displayTopics = (e, data) => {
     // handle your prevent defaults
     // set data dynamically on state.
 }`

File: src/components/pages/Home/RenderHomePage.js [hunk 5]
```diff
@@ -16,9 +16,6 @@ function RenderHomePage(props) {
     <Layout className="layout">
       <TopicsListContainer />
       {/* <TopicContextSlideoutContainer /> */}
```
Comment (ryan-hamblin) [inline src/components/pages/Home/RenderHomePage.js:18]: commented out code should be deleted.

File: src/styles/TopicDetails.css [hunk 6]
```diff
@@ -1,6 +1,33 @@
...
+
+.topicDetails__container h1{
+  font-size: xx-large;
+  margin-top: 5%;
+}
+
+.innerTopicDetails img{
+  width: 50px;
+  border-radius: 100px;
+}
```
Comment (ryan-hamblin) [inline src/styles/TopicDetails.css:29]: add a line break between your properties to keep things consistent

**Review by ryan-hamblin — COMMENTED** [review]
Please consider my changes to the formatting, remove logs/commented out code, and consider refactoring to improve the overall quality/maintainability of our codebase :)

**Pull request merged**